api/src/main/java/org/pmiops/workbench/actionaudit/ActionType.kt

```diff
@@ -10,5 +10,6 @@ enum class ActionType {
     DUPLICATE_FROM,
     DUPLICATE_TO,
     COLLABORATE,
-    DELETE
+    DELETE,
+    HIGH_EGRESS
```

Comment:
In reviewing this, I'm also reviewing hypothetical PRs in the future, and some of those are what's irking me. Not really fair to you I guess. I don't have the luxury of a real schema, so conventions are enforced by nagging.

What about `DETECT_EGRESS_VIOLATION`? It aligns narrowly with this anomaly type and is still a present tense verb that makes sense with the target. (For purity, `DETECT` would be the action and `VPC_EGRESS_VIOLATION` would be the target, but that won't work well because the workspace needs to be the target.)

The other consideration is how we treat any future system-detected security or policy flags against a user or workspace. If it's a separate action for each one, that's OK, but might mean more bespoke querying. In other words, by choosing an action that matches your classes, you're tipping the scale such that there are going to be more actions to manage, which many not be a net loss in cognitive overhead.

Reply:
I'm willing to bite on the idea that consistency around verbs vs. nouns is important. How about DETECT_HIGH_EGRESS_EVENT?

This is edging towards too many word-tokens for comfort, but I think we'll survive it.

Reply:
That works, and balances out consistency with your other names.